Use pregenerated C sources so Cython does not create extension modules
with hashes that differ from the generated sources.

Index: _setuputils.py
--- _setuputils.py.orig
+++ _setuputils.py
@@ -219,11 +219,8 @@ def _dummy_cythonize(extensions, **_kwar
         extension.sources[:] = sources
     return extensions
 
-try:
-    from Cython.Build import cythonize
-except ImportError:
-    # The .c files had better already exist.
-    cythonize = _dummy_cythonize
+# The .c files had better already exist.
+cythonize = _dummy_cythonize
 
 def cythonize1(ext):
     # All the directories we have .pxd files
